tests/unit/FmCorrection.spec.ts

```diff
@@ -6,7 +6,7 @@ import sinon from 'sinon'
 import { getVuexStore } from '@/store/'
```

Comment:
For now, this is mostly a copy of the original Correction.spec.ts page (with a few updates).

More to come in future commits...